packages/shared/src/components/Feed.tsx

```diff
@@ -199,14 +197,12 @@ export default function Feed<T>({
   const showScrollOnboardingVersion =
     sidebarRendered &&
     alerts?.filter &&
+    !user?.id &&
     Object.values(ScrollOnboardingVersion).includes(scrollOnboardingVersion);
-  const fetchPageFn = showScrollOnboardingVersion
-    ? () => setShowFeedFilters(true)
-    : fetchPage;
 
   const infiniteScrollRef = useFeedInfiniteScroll({
-    fetchPage: fetchPageFn,
-    canFetchMore,
+    fetchPage,
+    canFetchMore: !showScrollOnboardingVersion && canFetchMore,
```

Comment:
On `Control` they should be able to scroll, still.
```suggestion
    canFetchMore: (!showScrollOnboardingVersion || scrollOnboardingVersion === ScrollOnboardingVersion.Control) && canFetchMore,
```

Reply:
Just realised this from the Ido's comment. Will fix.

Reply:
Oh, just saw it. Yep, that should fix what he mentioned.